docs/configexamples/fwall-and-bridge.rst

```diff
@@ -0,0 +1,496 @@
+  set firewall ipv4 forward filter rule 110 description 'br1 traffic'
+  set firewall ipv4 forward filter rule 110 action 'jump'
+  set firewall ipv4 forward filter rule 110 inbound-interface group 'br1-ifaces'
+  set firewall ipv4 forward filter rule 110 jump-target 'ip-br1-fwd'
+
+  # Forward - Catch all traffic for br2
+  set firewall ipv4 forward filter rule 120 description 'br2 traffic'
+  set firewall ipv4 forward filter rule 120 action 'jump'
+  set firewall ipv4 forward filter rule 120 inbound-interface group 'br2-ifaces'
+  set firewall ipv4 forward filter rule 120 jump-target 'ip-br2-fwd'
+
+  # Forward - Default action drop:
+  set firewall ipv4 forward filter default-action 'drop'
+
+And content of the custom rule-sets:
+
+.. code-block:: none
+
+  ### br1 - ip-br1-fwd
+    # Requirement: Allow connections to internet
+  set firewall ipv4 name ip-br1-fwd rule 10 description 'br1 - allow internet access'
+  set firewall ipv4 name ip-br1-fwd rule 10 action 'accept'
+  set firewall ipv4 name ip-br1-fwd rule 10 outbound-interface name 'eth0'
+    # Requirement: Drop all other connections
+  set firewall ipv4 name ip-br1-fwd default-action 'drop'
+
+  ### br2 - ip-br2-fwd
+    # Requirement: Allow connections to internet
+  set firewall ipv4 name ip-br2-fwd rule 10 description 'br2 - allow internet access'
+  set firewall ipv4 name ip-br2-fwd rule 10 action 'accept'
+  set firewall ipv4 name ip-br2-fwd rule 10 outbound-interface name 'eth0'
+    # Requirement: Allow connections to br1
+  set firewall ipv4 name ip-br2-fwd rule 20 description 'br2 - allow access to br1'
+  set firewall ipv4 name ip-br2-fwd rule 20 action 'accept'
+  set firewall ipv4 name ip-br2-fwd rule 20 outbound-interface group 'br1-ifaces'
+    # Requirement: Drop all other connections
+  set firewall ipv4 name ip-br2-fwd default-action 'drop'
+
+
+Validation
+^^^^^^^^^^
+
+While testing the configuration, we can check logs in order to ensure that
+we are accepting and/or blocking the correct traffic.
+
+For example, while a host tries to get an IP address from a DHCP server in
+br1 all DHCP discover are dropped, and in br2, we can see that DHCP offers from
+untrusted servers are dropped:
+
+.. code-block:: none
+
+  vyos@bridge:~$ show log firewall bridge 
+  Sep 17 14:22:35 kernel: [bri-NAM-br2-fwd-22-D]IN=eth7 OUT=eth5 MAC=50:00:00:09:00:00:50:00:00:04:00:00:08:00 SRC=10.2.2.199 DST=10.2.2.92 LEN=322 TOS=0x10 PREC=0x00 TTL=128 ID=0 DF PROTO=UDP SPT=67 DPT=68 LEN=302 
+  Sep 17 14:28:18 kernel: [bri-NAM-br1-pre-10-D]IN=eth3 OUT= MAC=ff:ff:ff:ff:ff:ff:00:50:79:66:68:0c:08:00 SRC=0.0.0.0 DST=255.255.255.255 LEN=392 TOS=0x10 PREC=0x00 TTL=16 ID=0 PROTO=UDP SPT=68 DPT=67 LEN=372 
+  Sep 17 14:28:19 kernel: [bri-NAM-br1-pre-10-D]IN=eth3 OUT= MAC=ff:ff:ff:ff:ff:ff:00:50:79:66:68:0c:08:00 SRC=0.0.0.0 DST=255.255.255.255 LEN=392 TOS=0x10 PREC=0x00 TTL=16 ID=0 PROTO=UDP SPT=68 DPT=67 LEN=372 
+
+
+And with op-mode commands, we can check rules matchers, actions, and counters.
```

Comment:
```suggestion
And with operational mode commands, we can check rules matchers, actions, and counters.
```

I suppose we should avoid jargon in the docs. We know what "op-mode" means, but beginners probably don't.